README.md

```diff
@@ -1,3 +1,5 @@
 ### web and api test automation framework
 #### Tech Stack
-- Programming Language: Java
\ No newline at end of file
+- Programming Language: Java
+- Web Library : Selenium
+- API Library : Rest Assured
```

Comment:
This is a test review comment for single line

Reply:
Let me do the changes and come back